client/src/test/java/com/tencent/rss/client/impl/CheckBlockIdsBenchmark.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Tencent is pleased to support the open source community by making
+ * Firestorm-Spark remote shuffle server available.
+ *
+ * Copyright (C) 2021 THL A29 Limited, a Tencent company.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * https://opensource.org/licenses/Apache-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package com.tencent.rss.client.impl;
+
+import com.tencent.rss.common.util.RssUtils;
+import com.tencent.rss.storage.HdfsTestBase;
+
+import java.io.IOException;
+import java.util.Iterator;
+import java.util.concurrent.atomic.AtomicLong;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.roaringbitmap.longlong.Roaring64NavigableMap;
+import com.tencent.rss.client.util.ClientUtils;
+
+public class CheckBlockIdsBenchmark extends HdfsTestBase {
+  private static final String EXPECTED_EXCEPTION_MESSAGE = "Exception should be thrown";
+  private static AtomicLong ATOMIC_LONG = new AtomicLong(0);
+  @Rule
+  public ExpectedException thrown = ExpectedException.none();
+
+  @Test
```

Comment:
It shouldn't be mark as regular test case, it can be executed manually.

Reply:
Use normal main function instead.